decidim-core/app/helpers/decidim/replace_buttons_helper.rb

```diff
@@ -0,0 +1,41 @@
+module Decidim
+  module ReplaceButtonsHelper
+    # Overrides the submit tags to always be buttons instead of inputs.
+    # Buttons are much more stylable and less prone to bugs.
+    #
+    # value   - The text of the button
+    # options - Options to provide to the actual tag.
+    #
+    # Returns a SafeString with the tag.
+    def submit_tag(text = "Save changes", options = {})
```

Comment:
This is Rails' default, I didn't want to change the signature as it'd break existing libraries.